### DISCUSSION TITLE
append BLE sensor data to TelePeriod JSON

### DISCUSSION THREAD

**Discussion by bam80** [tony-fav/tasmota-blerry#82] [Q&A] [answered]
Hi, I wonder if this project could help me with the request:

> I need to combine two sensors tele data to a one MQTT JSON.
Currently, the data from BLE sensor is transmitted in a separate JSON (to the same topic as the other temp sensor, though).

> Also, I would like to publish particular sensor's data to two different topics.

https://github.com/arendst/Tasmota/discussions/20395

**Comment by tony-fav** [comment, marked as answer]
I think you should be able to get some inspiration from this project, sure. This project is basically a rule for collecting tasmota messages, some processing for extra data, then republishing how I wanted to. So, you need basically the same concept.
